scripts/utils.mjs

```diff
@@ -12,6 +12,10 @@ export async function getLatestBuiltFilesList() {
     `standard-notes-${version}-linux-x86_64.AppImage`,
     `standard-notes-${version}-linux-amd64.snap`,
 
+    `standard-notes-${version}-linux-arm64.AppImage`,
+    `standard-notes-${version}-linux-arm64.snap`,
+    // `standard-notes-${version}-linux-arm64.deb`,
```

Comment:
```suggestion
```

Reply:
It might make sense to also comment out the snap-line above that one for now, at least until I've found out more about the feasibility of creating the arm64 snap package on macOS – that way, one _could_ go forward with only the AppImage arm64 release for now, and maybe handle the snap release in a separate PR at a later point.
(This is me assuming that even a somewhat limited release for arm64 with only the AppImage would also be something worthwile :)

Reply:
Should be resolved with my latest commit.